### PR TITLE
Add capella-console-client

### PR DESCRIPTION
**Pull request by jordan-heemskerk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jordan-heemskerk** [comment]
@conda-forge/help-python I believe this is ready for review, thanks

File: recipes/capella-console-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f587b89c7a2fb37b85b7b9a0424acea500d7f82f5ff05f498cfabec55fc46c5a
+
+build:
```
Comment (BastianZim) [inline recipes/capella-console-client/meta.yaml:12]: Can this be noarch?

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of comments regarding the overall structure and then I'll have a look at the recipe in detail. 

Can you please also run the recipe through [Grayskull](https://github.com/conda-incubator/grayskull) and check some of the formatting?

File: recipes/capella-console-client/meta.yaml (hunk 1 above)
Comment (jordan-heemskerk) [reply recipes/capella-console-client/meta.yaml:12]: added

File: recipes/capella-console-client/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+requirements:
+  host:
+    - httpx >=0.18.2,<0.19.0
+    - poetry
+    - pip
+    - python
+    - python-dateutil >=2.8.2,<3.0.0
+    - retrying >=1.3.3,<2.0.0
+    - rich >=10.16.2,<11.0.0
```
[the remark this answers is not in the document]
Comment (jordan-heemskerk) [reply recipes/capella-console-client/meta.yaml]: Removed the unneeded host deps, oops

**Comment by jordan-heemskerk** [comment]
@BastianZim I ran grayskull and used the updated formatting

File: recipes/capella-console-client/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  entry_points:
+    - capella-console-wizard = capella_console_client.cli.wizard:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - poetry
```
Comment (BastianZim) [inline recipes/capella-console-client/meta.yaml:22]:
```suggestion
    - poetry-core >=1.0.0
```
Ref: https://github.com/capellaspace/console-client/blob/bad631f207043231630c9b8c0893a1b4382d4061/pyproject.toml#L64

**Review by BastianZim — CHANGES_REQUESTED** [review]
Looks good. Just one small thing and then this should be ready to go!

**Pull request merged**